Dockerfile

```diff
@@ -129,8 +129,13 @@ RUN . /opt/ros/lcas/install/setup.sh && \
 RUN cd /opt/ros/lcas && colcon build && \
     rm -rf /opt/ros/lcas/src/ /opt/ros/lcas/build/ /opt/ros/lcas/log/
 
+# Install code-server
 RUN curl -fsSL https://code-server.dev/install.sh | sh
 
-USER ros
```

Comment:
why has this been removed? This changes the final image as it leaves it with root user. Is it intentional?